recipes/pdsspect/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pdsspect" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "c5207e99bb3131579b18290430390f60373758d98e59e3781c6e30b01ead5359" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - ginga 2.6.0
```

Comment:
It doesn't look like `ginga` is required to build the package. Please check and if this is the case remove it from the `build` requirements.